pjsip/include/pjsip-ua/sip_siprec.h

```diff
@@ -0,0 +1,135 @@
+#ifndef __PJSIP_SIPREC_H__
+#define __PJSIP_SIPREC_H__
+
+/**
+ * @file sip_siprec.h
+ * @brief SIP Session Recording Protocol (siprec) support (RFC 7866 - Session Recording Protocol in SIP)
+ */
+
+
+#include <pjsip-ua/sip_inv.h>
+#include <pjsip/sip_msg.h>
+
+/**
+ * @defgroup PJSIP_SIPREC SIP Session Recording Protocol (siprec) support (RFC 7866 - Session Recording Protocol in SIP)
+ * @brief SIP Session Recording Protocol support (RFC 7866 - Session Recording Protocol in SIP)
+ * @{
+ *
+ * \section PJSIP_SIPREC_REFERENCE References
+ *
+ * References:
+ *  - <A HREF="http://www.ietf.org/rfc/rfc7866.txt">RFC 7866: Session Recording Protocol (siprec)
+ *    in the Session Initiation Protocol (SIP)</A>
+ */
+PJ_BEGIN_DECL
+
+/**
+ * Initialize siprec module. This function must be called once during
+ * application initialization, to register siprec module to SIP endpoint.
+ *
+ * @param endpt         The SIP endpoint instance.
+ *
+ * @return              PJ_SUCCESS if module is successfully initialized.
+ */
+PJ_DECL(pj_status_t) pjsip_siprec_init_module(pjsip_endpoint *endpt);
+
+
+/**
+ * Create a=label attribute.
+ *
+ * @param media          The SDP media.
+ *
+ * @return               SDP label attribute.
+ */
+PJ_DECL(pjmedia_sdp_attr*) pjmedia_sdp_attr_get_label(pjmedia_sdp_media *sdp_media);
```

Comment:
This does not look right, a PJMEDIA SDP API in PJSIP, should remove this or move it to PJMEDIA SDP.
The implementation looks simple/one-liner, so I think you can just use the generic API `pjmedia_sdp_media_find_attr()` directly without this API.

Reply:
I removed the pjmedia_sdp_attr_get_label function, but I am still using the pjmedia_sdp_media_find_attr function siprec.c